fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.json

```diff
@@ -294,21 +294,25 @@
    "print": 1,
    "read": 1,
    "report": 1,
-   "role": "System Manager",
+   "role": "Administrator",
    "share": 1,
    "write": 1
   },
   {
-   "create": 1,
+   "delete": 1,
    "email": 1,
    "export": 1,
-   "print": 1,
    "read": 1,
    "report": 1,
-   "role": "All",
-   "select": 1,
-   "share": 1,
-   "write": 1
+   "role": "System Manager"
+  },
+  {
+   "read": 1,
+   "role": "Guest"
```

Comment:
does this mean guests can read a profile online but not do a get all?

Reply:
They can view the details of a profile. The "Select" mostly handles the `get_list` sort of things. Should we disable Read as well for a good measure?

Reply:
> The "Select" mostly handles the get_list sort of things.

okay. so with "read", an API call can be made to get a specific users' information. right? whether that API call happens via the website or via the CLI is irrelevant. the username is publicly visible on our website in many places.

Reply:
Yes. I enabled read because profiles have web views. Should we disable read for now and find alternative ways to show information on webpage? or alternatively try to find ways to hide critical information better